### DISCUSSION TITLE
File search filters

### DISCUSSION THREAD

**Discussion by justrealmilk** [directus/directus#7239] [Feature Requests]
![image](https://user-images.githubusercontent.com/156681/128477404-1850fa3a-f76e-4e4e-b2f2-fc3b57c9048a.png)

So, I have a few images (6,988) and I'm absolutely pumped to see folders get front seat support in version 9 but filtering down to a folder (or other meta) in the file selection interface isn't yet available. 

It'd obviously be a great addition. I hope this isn't a duplicate.

_Either way, once I'm on top of my migration I'll be considering a PR if no one beats me to it_

**Comment by rijkvanzanten** [comment]
This is a big start for that! https://github.com/directus/directus/pull/7036

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
